Sources/ComposableArchitecture/SwiftUI/ActionSheet.swift

```diff
@@ -127,23 +128,44 @@ public struct ActionSheetState<Action> {
 @available(macOS, unavailable)
 @available(tvOS 13, *)
 @available(watchOS 6, *)
-extension ActionSheetState: Equatable where Action: Equatable {}
+extension ActionSheetState: Equatable where Action: Equatable {
+  public static func == (lhs: Self, rhs: Self) -> Bool {
```

Comment:
Have to do custom implementations of `Equatable`, `Hashable`, and `CustomDebugOutputConvertible` now to avoid the `UUID` being different in tests...maybe you have a better idea than hiding a UUID inside?